gc/stats/ScavengerCopyScanRatio.hpp

```diff
@@ -121,9 +123,12 @@ class MM_ScavengerCopyScanRatio
 	uintptr_t _threadCount;						/**< number of gc threads participating in current gc cycle */
 	uintptr_t _historyFoldingFactor;			/** number of major updates per history record */
 	uintptr_t _historyTableIndex;				/** index of history table record that will receive next major update */
-	UpdateHistory _historyTable[SCAVENGER_UPDATE_HISTORY_SIZE];
-
+	
+	/** Snapshot of Scan List metrics taken during minor update, used in the case where the update would of triggered a major update had it reached the threshold, but doesn't, and requires flushing */
+	uintptr_t nonEmptyScanListsForFlush;
+	uintptr_t cachesQueuedFlushCacheForFlush;
```

Comment:
this class had '_' prepended to field names. keep it consistent

Reply:
Removed snapshot/caching completely